source/deploy/consider-mobile-vpn-options.rst

```diff
@@ -8,7 +8,7 @@ You need to set up a way to connect to your private network Mattermost instance,
 
 Depending on your security policies, we recommend deploying Mattermost behind a VPN and using a `per-app VPN <#id3>`_ with your EMM provider, or a mobile VPN client.
 
-Also consider deploying a mobile VPN client with multi-factor authentication (MFA) to your preferred login method, such as GitLab SSO with MFA, or run Mattermost Enterprise Edition with `multi-factor authentication (MFA) <https://docs.mattermost.com/deployment/auth.html>`__ enabled.
+Also consider deploying a mobile VPN client with multi-factor authentication (MFA) to your preferred login method, such as GitLab SSO with MFA, or run Mattermost Enterprise Edition with `multi-factor authentication (MFA) <https://docs.mattermost.com/onboard/multi-factor-authentication.html>`__ enabled.
```

Comment:
This link has a 301 redirect and is already going to the correct page. :)

Reply:
Can you help me understand what is 301 redirect? @justinegeffen . It will help me in future.

Reply:
It's a way to make sure older pages go to the new ones automatically. It's not part of the process that you need to follow though! :) And we have discussed this change and it works for us. :)

Reply:
Okay okay thanks a lot